<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainLayout</class>
 <widget class="QDialog" name="MainLayout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1193</width>
    <height>126</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1193</width>
    <height>126</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1193</width>
    <height>126</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MahjongQT</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="2">
    <layout class="QHBoxLayout" name="MahjongLayout">
     <property name="leftMargin">
      <number>7</number>
     </property>
     <property name="topMargin">
      <number>7</number>
     </property>
     <property name="rightMargin">
      <number>7</number>
     </property>
     <property name="bottomMargin">
      <number>7</number>
     </property>
     <item>
      <widget class="QLabel" name="Mahjong_1">
       <property name="minimumSize">
        <size>
         <width>69</width>
         <height>90</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>69</width>
         <height>90</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Mahjong_2">
       <property name="minimumSize">
        <size>
         <width>69</width>
         <height>90</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>69</width>
         <height>90</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Mahjong_3">
       <property name="minimumSize">
        <size>
         <width>69</width>
         <height>90</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>69</width>
         <height>90</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Mahjong_4">
       <property name="minimumSize">
        <size>
         <width>69</width>
         <height>90</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>69</width>
         <height>90</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Mahjong_5">
       <property name="minimumSize">
        <size>
         <width>69</width>
         <height>90</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>69</width>
         <height>90</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Mahjong_6">
       <property name="minimumSize">
        <size>
         <width>69</width>
         <height>90</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>69</width>
         <height>90</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Mahjong_7">
       <property name="minimumSize">
        <size>
         <width>69</width>
         <height>90</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>69</width>
         <height>90</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Mahjong_8">
       <property name="minimumSize">
        <size>
         <width>69</width>
         <height>90</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>69</width>
         <height>90</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Mahjong_9">
       <property name="minimumSize">
        <size>
         <width>69</width>
         <height>90</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>69</width>
         <height>90</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Mahjong_10">
       <property name="minimumSize">
        <size>
         <width>69</width>
         <height>90</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>69</width>
         <height>90</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Mahjong_11">
       <property name="minimumSize">
        <size>
         <width>69</width>
         <height>90</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>69</width>
         <height>90</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Mahjong_12">
       <property name="minimumSize">
        <size>
         <width>69</width>
         <height>90</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>69</width>
         <height>90</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Mahjong_13">
       <property name="minimumSize">
        <size>
         <width>69</width>
         <height>90</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>69</width>
         <height>90</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="btnShuffle">
     <property name="minimumSize">
      <size>
       <width>93</width>
       <height>28</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>93</width>
       <height>28</height>
      </size>
     </property>
     <property name="text">
      <string>Shuffle(&amp;S)</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="btnClose">
     <property name="minimumSize">
      <size>
       <width>93</width>
       <height>28</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>93</width>
       <height>28</height>
      </size>
     </property>
     <property name="text">
      <string>Close(&amp;C)</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MahjongQT.qrc"/>
 </resources>
 <connections/>
</ui>
